jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/Etcd.java

```diff
@@ -58,6 +58,7 @@ public static class Builder {
         private List<String> additionalArgs;
         private Network network;
         private boolean shouldMountDataDirectory = true;
+        private String user = "";
```

Comment:
I think you could leave it as `null`

Reply:
- I changed the default value to `null` in the new commit. The decision logic for Linux has been removed, although I don't know what happens on Windows/Mac.